File: cleanup_admin.js
const fs = require('fs');
const path = 'c:\\Users\\usuário\\Documents\\Projeto Car\\client\\src\\pages\\admin.tsx';

try {
    const content = fs.readFileSync(path, 'utf8');
    const lines = content.split('\n');

    // Lines were 1-indexed in view_file.
    // We want to keep 1 to 170 (index 0 to 169)
    // And keep 6471 to end (index 6470 to end)
    const newLines = [...lines.slice(0, 170), ...lines.slice(6470)];

    fs.writeFileSync(path, newLines.join('\n'), 'utf8');
    console.log('File cleaned up successfully with Node.js');
} catch (err) {
    console.error('Error cleaning up file:', err);
}

File: cleanup_admin.py
import os

file_path = r'c:\Users\usuário\Documents\Projeto Car\client\src\pages\admin.tsx'

with open(file_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# VendorWorkspace was inserted at line 55 (index 54).
# My new version ends at index 169 (line 170).
# The mess starts at index 170 (line 171) and ends at line 6470 (index 6469).

new_lines = lines[:170] + lines[6470:]

with open(file_path, 'w', encoding='utf-8') as f:
    f.writelines(new_lines)

print("File cleaned up successfully.")
